scuffle-ffmpeg doesn't compile on windows

### What seems to be the problem?

My code has not even been modified from the base hello world program and it still doesn't compile.
```
D:\PetProjects\ffmpreg> cargo build -j(nproc.exe --all)
   Compiling nutype-enum v0.1.0 (https://github.com/ScuffleCloud/scuffle#26d9eb13)
   Compiling scuffle-ffmpeg v0.1.2 (https://github.com/ScuffleCloud/scuffle#26d9eb13)
error[E0412]: cannot find type `__va_list_tag` in this scope
  --> C:\Users\thebe\scoop\persist\rustup\.cargo\git\checkouts\scuffle-fa5454f3849b2d06\26d9eb1\crates\ffmpeg\src\log.rs:87:108
   |
87 | ...st libc::c_char, va: *mut __va_list_tag) {
   |                              ^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `SWS_BILINEAR` in this scope
  --> C:\Users\thebe\scoop\persist\rustup\.cargo\git\checkouts\scuffle-fa5454f3849b2d06\26d9eb1\crates\ffmpeg\src\scaler.rs:38:17
   |
38 |                 SWS_BILINEAR,
   |                 ^^^^^^^^^^^^ not found in this scope
   |
help: consider importing one of these unit variants
   |
1  + use crate::scaler::SwsFlags::SWS_BILINEAR;
   |
1  + use ffmpeg_sys_next::SwsFlags::SWS_BILINEAR;
   |

error[E0425]: cannot find function `vsnprintf` in this scope
     --> C:\Users\thebe\scoop\persist\rustup\.cargo\git\checkouts\scuffle-fa5454f3849b2d06\26d9eb1\crates\ffmpeg\src\log.rs:100:5
      |
100   |     vsnprintf(buf.as_mut_ptr() as *mut i8, buf.len() as _, fmt, va);
      |     ^^^^^^^^^ help: a function with a similar name exists: `av_bprintf`
      |
     ::: D:\PetProjects\ffmpreg\target\debug\build\ffmpeg-sys-next-9c98127f5e289506\out/bindings.rs:12406:5
      |
12406 |     pub fn av_bprintf(buf: *mut AVBPrint, fmt: *const libc::c_char, ...);
      |     --------------------------------------------------------------------- similarly named function `av_bprintf` defined here

Some errors have detailed explanations: E0412, E0425.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `scuffle-ffmpeg` (lib) due to 3 previous errors
```

### Relevant log output

```shell
N/A
```

--- a/crates/ffmpeg/CHANGELOG.md
+++ b/crates/ffmpeg/CHANGELOG.md
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Fixed windows build (#238)
+
 ## [0.1.2](https://github.com/ScuffleCloud/scuffle/compare/scuffle-ffmpeg-v0.1.1...scuffle-ffmpeg-v0.1.2) - 2025-02-10
 
 ## [0.1.1](https://github.com/ScuffleCloud/scuffle/compare/scuffle-ffmpeg-v0.1.0...scuffle-ffmpeg-v0.1.1) - 2025-02-10

--- a/crates/ffmpeg/src/log.rs
+++ b/crates/ffmpeg/src/log.rs
@@ -84,7 +84,18 @@ pub fn log_callback_unset() {
     }
 }
 
-unsafe extern "C" fn log_cb(ptr: *mut libc::c_void, level: libc::c_int, fmt: *const libc::c_char, va: *mut __va_list_tag) {
+#[cfg(unix)]
+type VaList = *mut __va_list_tag;
+
+#[cfg(windows)]
+type VaList = va_list;
+
+#[cfg(windows)]
+extern "C" {
+    fn vsnprintf(buffer: *mut libc::c_char, count: libc::size_t, format: *const libc::c_char, ap: VaList) -> i32;
+}
+
+unsafe extern "C" fn log_cb(ptr: *mut libc::c_void, level: libc::c_int, fmt: *const libc::c_char, va: VaList) {
     let level = LogLevel::from(level);
     let class = NonNull::new(ptr as *mut *mut AVClass)
         .and_then(|class| NonNull::new(*class.as_ptr()))

--- a/crates/ffmpeg/src/scaler.rs
+++ b/crates/ffmpeg/src/scaler.rs
@@ -1,4 +1,6 @@
 use ffmpeg_sys_next::*;
+#[cfg(windows)]
+use SwsFlags::*;
 
 use crate::error::{FfmpegError, FfmpegErrorCode};
 use crate::frame::{Frame, VideoFrame};
@@ -35,7 +37,7 @@ impl Scaler {
                 width,
                 height,
                 pixel_format,
-                SWS_BILINEAR,
+                SWS_BILINEAR as i32,
                 std::ptr::null_mut(),
                 std::ptr::null_mut(),
                 std::ptr::null(),

